narratives/ncov_sit-rep_2020-04-10.md

```diff
@@ -0,0 +1,270 @@
+<br><br>
+This indicates that while we hadn't yet sequenced any samples from this clade, the origins of the European outbreak were already cryptically circulating by mid-January.
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Cryptic transmission in North America from mid-January](https://nextstrain.org/ncov/global/2020-04-09?d=tree,map&dmax=2020-02-14&label=clade:B1&p=full)
+
+Similarly, zooming in to the bottom of the tree, we see an ancestral North American virus (internal node) that initiated what would later become a large outbreak in North America.
+<br><br>
+This indicates that although we only had 1 sequenced case from this cluster at the time, the virus was circulating cryptically within North America by mid-January.
+
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Expansion of North American and European outbreaks](https://nextstrain.org/ncov/global/2020-04-09?dmax=2020-03-15&dmin=2020-02-15&p=full&d=tree,map)
+
+During the 3rd month of the outbreak, these two clusters expanded to cause large outbreaks within Europe and North America, respectively.
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Spread across international borders](https://nextstrain.org/ncov/global/2020-04-09?dmax=2020-03-15&dmin=2020-02-15&f_region=North%20America&p=full&r=division&d=tree,map)
+
+Within each of these regions, we see widespread mixing across state lines and international borders.
+This is demonstrated here by the intermingling of samples from multiple U.S. states and Canadian provinces.
+<br><br>
+We've previously reported on a similar pattern of frequent mixing of viral lineages between European countries.
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Introductions from North America & Europe to other parts of the globe](https://nextstrain.org/ncov/global/2020-04-09?dmax=2020-03-15&dmin=2020-02-15&f_region=Africa,South%20America,Oceania&p=full&d=tree,map)
+
+These clusters are not restricted to their respective regions. During this time period, we also see cases from South America, Oceania, and Africa against a backdrop of primarily European and North American clades. This indicates that there were frequent intercontinental introductions during this time period.
+<br><br>
+Interestingly, samples from the global south span the full breadth of the tree, indicating that these regions' outbreaks are mixtures of many viral lineages.
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Coming full circle: reintroductions to Asia](https://nextstrain.org/ncov/global/2020-04-09?branchLabel=clade&d=tree,map&dmin=2020-03-14&f_region=Asia&p=full&r=division)
```

Comment:
Please remove the `branchLabel` query here (unless there's a specific reason it's included which i'm missing)